Documentation/compatibility/winforms-accessibility-changes-48.md

```diff
@@ -0,0 +1,105 @@
+      </runtime>
+    </configuration>
+```    
+Note that to opt in to the accessibility features added in .NET Framework 4.8, you must also opt in to accessibility features of .NET Framework 4.7.1 and 4.7.2 as well.
+Applications that target the .NET Framework 4.8 and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to `true`.
+
+__Use of OS-defined colors in High Contrast themes__
+// To be filled.
+
+__Improved Narrator support__
+- Narrator now announces the sort direction of the <xref:System.Windows.Forms.DataGridViewColumn> when announcing an accessible name of a <xref:System.Windows.Forms.DataGridViewCell>. 
+
+__Improved Checked ListBox Accessibility support__
+- Improved Narrator support for <xref:System.Windows.Forms.CheckedListBox> control. At this moment when navigating to the <xref:System.Windows.Forms.CheckedListBox> control using keyboard, Narrator focuses the <xref:System.Windows.Forms.CheckedListBox> item and announces it.
+
+__Improved ComboBox Accessibility support__
+- Enabled UI Automation support for <xref:System.Windows.Forms.ComboBox> control with ability to use UI Automation notifications and other UI Automation features.
+
+__Improved DataGridView Accessibility support__
+- Enabled UI Automation support for <xref:System.Windows.Forms.DataGridView> control with ability to use UI Automation notifications and other UI Automation features.
+- The UI Automation element which corresponds to the <xref:System.Windows.Forms.DataGridViewComboBoxEditingControl> or <xref:System.Windows.Forms.DataGridViewTextBoxEditingControl> is now a child of corresponding editing cell.
+
+__Improved Link Label Accessibility support__
+- Improved <xref:System.Windows.Forms.LinkLabel> control accessibility: Narrator announces the disabled state for the link if corresponding <xref:System.Windows.Forms.LinkLabel> control is disabled.
+
+__Improved Progress Bar Accessibility support__
+- Enabled UI Automation support for <xref:System.Windows.Forms.ProgressBar> control with ability to use UI Automation notifications and other UI Automation features. Developers are now able to use UI Automation notifications which Narrator can announce to indicate the progress.
+
+__Improved Property Grid Accessibility support__
+- Enabled UI Automation support for <xref:System.Windows.Forms.PropertyGrid> control with ability to use UI Automation notifications and other UI Automation features.
+- The UI Automation element which corresponds to the currently editing property is now a child of corresponding property item UI Automation element.
+- The UI Automation property item element is now a child of corresponding category if the parent <xref:System.Windows.Forms.PropertyGrid> control is set to Category view.
```

Comment:
```suggestion
- The UI Automation property item element is now a child of the corresponding category element if the parent <xref:System.Windows.Forms.PropertyGrid> control is set to category view.
```
I'm not sure about capitalization of the word `category`.

Reply:
Thanks. I agree, category should not have 'c' capitalized.